filebeat/module/postgresql/log/test/postgresql-9.6-debian-with-slowlog.log-expected.json

```diff
@@ -10,12 +10,12 @@
         "log.level": "LOG",
         "log.offset": 0,
         "message": "database system was shut down at 2017-06-17 16:58:04 CEST",
-        "postgresql.log.timestamp": "2017-07-31 13:36:42.585",
+        "postgresql.log.timestamp": "2017-07-31 13:36:42.585 CEST",
```

Comment:
Sounds like a field we should deprecate?

Reply:
Yes, I have marked it as deprecated.